### DISCUSSION TITLE
shop.image_url_760x100 always null?

### DISCUSSION THREAD

**Discussion by MickaelBNeron** [etsy/open-api#986] [Q&A]
Whenever I call

GET https://openapi.etsy.com/v3/application/users/{user_id}/shops

or

GET https://openapi.etsy.com/v3/application/shops/{shop_id}

the field image_url_760x100 is always null (even though my shop has a banner). the icon_url_fullxfull property has the correct URL.

Anyone tried either of these GET requests and properly obtained the banner's URL?

**Comment by Gareth064** [comment]
I can confirm that this also happens for me too. It is returning as Null

to be fair, I am using getShopByOwnerUserId here but its the same Shop object returned.

![image](https://user-images.githubusercontent.com/22307223/236177951-bb64414a-8ac9-4751-8a95-0139c9e5df4e.png)

**Reply by Gareth064** [reply]
Same result when using "findShops" as well.

@MickaelBNeron  Can you email developer@etsy.com and ask them to investigate? Link them to this GitHub discussion for reference.

**Reply by MickaelBNeron** [reply]
Sure
